<!--  -->
<launch>
  <node pkg="mrop" type="manual_controller" name="manual_controller" output="screen">
     
    <param name="manual_controller_active"   value="true"/>
    <param name="use_keys"                   value="true"/>
    <param name="use_joy"                    value="true"/>

    <param name="linear_gain"                value="0.0"/>
    <param name="angular_gain"               value="0.0"/>
  </node>
</launch>

